src/analyze/analyze.c

```diff
@@ -2366,6 +2382,18 @@ static int parse_argv(int argc, char *argv[]) {
 
                         break;
 
+                case ARG_ALIAS: ;
+                        if (argc > 4)
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Invalid argument count for --alias");
```

Comment:
Might want to be more specific here - eg, say clearly that only one unit can be passed